pyconca2017/templates/pages/volunteer.html

```diff
@@ -5,15 +5,15 @@
 
   <div class="container">
     <div class="row">
-      <div class="col-md-12">
+      <div class="col-md-12 c-block-text">
         <h1>{% trans "Volunteering at PyCon Canada 2017" %}</h1>
-        <p>
-          {% blocktrans %}
-            We're all looking forward to the great conference in November -the organizing team is working hard to make this happen, but we can't do it alone.
-
-            Volunteers <strong>must be registered</strong> to volunteer, but we welcome <strong>all attendees</strong> to lend a hand. We much appreciate any assistance you can give us.
-          {% endblocktrans %}
-        </p>
+          <p>{% trans "We're all looking forward to the great conference in November. The organizing team is working hard to make this happen, but we can't do it alone." %}</p>
+          <p>{% trans "Volunteers <strong>must be registered</strong> to volunteer, but we welcome <strong>all attendees</strong> to lend a hand. We appreciate any assistance you can give us." %}</p>
+          <p>
+          <a class="c-btn c-btn--primary" href="https://docs.google.com/a/ectobal.com/forms/d/1Iuh4BaDtkhF8shr3HnnKhFbjNOy0ZwRzxoiCv0AywoI/edit?usp=sharing" >
```

Comment:
Let's use this link instead: 
https://docs.google.com/forms/d/1Iuh4BaDtkhF8shr3HnnKhFbjNOy0ZwRzxoiCv0AywoI/edit